### PR TITLE
PSF ellipticity metric

### PR DESCRIPTION
**Pull request by danjampro**
Adds the PSF ellipticity as a calexp-stage metric

### CODE REVIEW

File: src/huntsman/drp/metrics/calexp.py [hunk 1]
```diff
@@ -1,31 +1,65 @@
...
+        dict: A dictionary of metric name: value.
+    """
+    result = {}
+    for metric in METRICS:
+        func = load_module(f"huntsman.drp.metrics.calexp.{metric}")
+        result.update(func(calexp))
+    return result
+
+
+def zeropoint(calexp):
```
Comment (lspitler) [inline src/huntsman/drp/metrics/calexp.py:30]:
```suggestion
def get_zeropoint(calexp):
```

File: src/huntsman/drp/metrics/calexp.py [hunk 2]
```diff
@@ -1,31 +1,65 @@
...
     """
     fluxzero = calexp.getPhotoCalib().getInstFluxAtZeroMagnitude()
     # Note the missing minus sign here...
-    return 2.5 * np.log10(fluxzero) * u.mag
+    return {"zp_mag": 2.5 * np.log10(fluxzero) * u.mag}
 
 
-def psf_fwhm(calexp):
-    """ Calculate the PSF FWHM.
+def psf(calexp):
```
Comment (lspitler) [inline src/huntsman/drp/metrics/calexp.py:42]:
```suggestion
def get_psf_fwhm(calexp):
```

File: src/huntsman/drp/metrics/calexp.py [hunk 3]
```diff
@@ -1,31 +1,65 @@
...
+def zeropoint(calexp):
     """ Get the magnitude zero point of the raw data.
     Args:
         calexp (lsst.afw.image.exposure): The calexp object.
     Returns:
-        astropy.Quantity: The magnitude zero point.
+        dict: Dict containing the zeropoint in mags.
     """
     fluxzero = calexp.getPhotoCalib().getInstFluxAtZeroMagnitude()
     # Note the missing minus sign here...
```
Comment (lspitler) [inline src/huntsman/drp/metrics/calexp.py:38]: why no negative?

File: src/huntsman/drp/metrics/calexp.py [hunk 4]
```diff
@@ -1,31 +1,65 @@
...
-    return 2.5 * np.log10(fluxzero) * u.mag
+    return {"zp_mag": 2.5 * np.log10(fluxzero) * u.mag}
```
Comment (lspitler) [inline src/huntsman/drp/metrics/calexp.py:39]: Is this `u.ABmag`?

**Review by lspitler — APPROVED** [review]
A few suggestions, but you probably have good reasons for doing most...

File: src/huntsman/drp/metrics/calexp.py (hunk 3 above)
Comment (danjampro) [reply src/huntsman/drp/metrics/calexp.py:38]: Not exactly sure, but this seems to give the same result as printed in the LSST logs.

File: src/huntsman/drp/metrics/calexp.py (hunk 4 above)
Comment (danjampro) [reply src/huntsman/drp/metrics/calexp.py:39]: I guess it is in whatever units the reference catalogue is. So yes I would expect it to be in AB mag.

File: src/huntsman/drp/metrics/calexp.py (hunk 2 above)
Comment (danjampro) [reply src/huntsman/drp/metrics/calexp.py:42]: Well, we also get the ellipticity

**Pull request merged**